src/lib/db/api-token-store.ts

```diff
@@ -200,7 +200,7 @@ export class ApiTokenStore implements IApiTokenStore {
         const now = new Date();
         try {
             await this.db(TABLE)
-                .whereIn('secrets', secrets)
+                .whereIn('secret', secrets)
```

Comment:
Is this a bug unrelated to this PR? Just curious

Reply:
Yep, this `markSeenAt` method was already implemented from a while ago, git says March 2021, except it was not being used AFAIK and had this typo. So I guess I'm following the approach that was originally thought of, with the `secrets string[]` parameter and all.

Reply:
nice catch! 🗡️